src/reducers/rootReducer.ts

```diff
@@ -0,0 +1,26 @@
+import { InitialState, Action } from '../types/default';
+import i18n from '../i18n';
+
+const initState = {
+  test: 'test',
+  theme: 'default',
+  lang: 'en',
+};
+
+const rootReducer = (state = initState, action: Action): InitialState => {
+  switch (action.type) {
+    case 'CHANGE_THEME':
+      state.theme = state.theme === 'default' ? 'dark' : 'default';
+      return state;
+      break;
+    case 'CHANGE_LNG':
+      state.lang = action.payload;
+      i18n.changeLanguage(action.payload);
```

Comment:
please put and store this value in the Redux Store. 

Create redux action `changeLanguage` or smth like that